fetch.bs

```diff
@@ -4399,15 +4402,23 @@ steps:
 
  <li><p>If <var>timingInfo</var> is null, then return.
 
- <li>
-  <p>If <var>response</var>'s <a for=response>timing allow passed flag</a> is not set, then:
+ <li><p>Let <var>taoStatus</var> be <code>success</code>.
```

Comment:
Can we make this "Let taoSucceeded be true" instead? That seems preferable to introducing new types.